application/src/main/java/org/thingsboard/server/service/subscription/SubscriptionManagerService.java

```diff
@@ -48,5 +51,8 @@ public interface SubscriptionManagerService extends ApplicationListener<Partitio
 
     void onAlarmDeleted(TenantId tenantId, EntityId entityId, AlarmInfo alarm, TbCallback callback);
 
+    void onNotificationUpdate(TenantId tenantId, UserId recipientId, NotificationUpdate notificationUpdate, TbCallback callback);
```

Comment:
You don't need this anymore, right?

Reply:
It is needed in case user marks notification as read